doc/ref_model/chapters/chapter08.md

```diff
@@ -98,8 +98,78 @@ The Cloud Services and the Cloud Resources Brokers provide value-added services
+- Manage App/VNF - Image / Container / Registry Management, Deploy/Configure/Scale/Start/Stop App/VNF, Monitor App/VNFs
+- Transactions / Conversations - Use Communications Services, Use Edge Applications Services, Use Cloud Services
+
+<a name="8.2.4.2"></a>
+### 8.2.4.2 Stereo-Typical Scenarios
+A set of stereo-typical interactions cases are illustrated for the cases of a Simple Infrastructure-as-a-Service (IaaS) and Software-as-a-Service (SaaS), where deployment is on a Cloud Provider's centralised sites and/or Edge sites. The scenarios help highlight needs for the Cloud Service Broker and Cloud Resources Broker (as per Figure 8.3) and hence extent of orchestration required to manage the interactions.
+
+<p align="center"><img src="../figures/rm-chap8-multi-cloud-interactions-simple-stereo-types-02.png" alt="Simple Stereo-Typical Interactions"></p>
+<p align="center"><b>Figure 8-5:</b> "Simple Stereo-Typical Interactions"</p>
+
+The following patterns are visible:
+- For IaaS Cloud Integration:
+  - Cloud behaves like a set of virtual servers and, thus, requires virtual server life-cycle management and orchestration
+  - Depending on whether the cloud is accessed via public internet or private connection will change the extend of the Connectivity Management
+- For SaaS Cloud Integration:
+  - Cloud behaves like a running application/service and requires subscription management, and complex orchestration of the app/service and underlying resources is managed by SaaS provider with the User is relieved of having to provide direct control of resources
+- For CaaS Cloud Integration:
+  - Registry for pulling Containers could be from:
+    - Cloud in which case consumption model is closer to SaaS or
+    - from Private / Public Registry in which case integration model requires specific registry management elements
+- For Edge Cloud Integration:
+  - Adds need for Communications Service Provider and Cloud Provider physical, network underlay and overlay connectivity management
+
+A disaggregated scenario for a CSP using SaaS who uses IaaS is illustrated in the following diagram:
+
+<p align="center"><img src="../figures/rm-chap8-multi-cloud-interactions-disaggregated-stereo-type-01.png" alt="Disaggregated SaaS Stereo-Typical Interaction"></p>
+<p align="center"><b>Figure 8-5:</b> "Disaggregated SaaS Stereo-Typical Interaction"</p>
 
+In disaggregated SaaS scenario the application provider is able to operate as an "infra-structureless" organisation. This could be achieved through SaaS organisation using public IaaS Cloud Providers which could include the CSP itself. A key consideration for CSP in both cloud provision and consumption in Multi-Cloud scenario is how to manage the integration across the Cloud Providers.
+
+To make this manageeable and avoid having to manage integration there are a number of models:
+- Industry Standard APIs that allow consistent consumption across Cloud Providers,
```

Comment:
Which agreed Industry Standard APIs exist for that, or which SDO works on it and has chance to get adoption?